### PR TITLE
feat: add question flow for importing existing SPFx solution to TTK

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
ADO: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24088108
Screenshot:
image

### CODE REVIEW

**Review by sffamily — DISMISSED** [review]

**Review by yukun-dong — APPROVED** [review]

File: packages/fx-core/src/component/generator/spfx/utils/questions.ts [hunk 1]
```diff
@@ -117,3 +121,36 @@ export const spfxPackageSelectQuestion: Question = {
...
+    name: SPFXQuestionNames.spfx_import_folder,
+    title: getLocalizedString("core.spfxFolder.title"),
+    placeholder: getLocalizedString("core.spfxFolder.placeholder"),
+  };
+}
+
+export const skipAppName: Question = {
+  type: "func",
+  name: SPFXQuestionNames.skip_app_name,
+  title: "Set app name to skip",
```
Comment (tecton) [inline packages/fx-core/src/component/generator/spfx/utils/questions.ts:150]: Does this one need to be localized?

File: packages/fx-core/tests/core/FxCore.create.test.ts [hunk 2]
```diff
@@ -106,6 +106,24 @@ describe("Core basic APIs for v3", () => {
...
+      stage: Stage.create,
+      [CoreQuestionNames.Capabilities]: [TabSPFxItem().id],
+      [CoreQuestionNames.ProgrammingLanguage]: "typescript",
+      [SPFXQuestionNames.spfx_solution]: "import",
+      [SPFXQuestionNames.spfx_import_folder]: "c:\\test",
+    };
+    const core = new FxCore(tools);
+    const res = await core.createProject(inputs);
+    assert.isTrue(res.isOk());
+    inputs[CoreQuestionNames.AppName] = "appnamePlaceholder";
```
Comment (tecton) [inline packages/fx-core/tests/core/FxCore.create.test.ts:124]: Is this an assertion?

**Review by jayzhang — DISMISSED** [review]

File: packages/fx-core/src/component/generator/spfx/utils/questions.ts (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/src/component/generator/spfx/utils/questions.ts:140]: This is an invisible question so no need to localize

File: packages/fx-core/tests/core/FxCore.create.test.ts (hunk 2 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/tests/core/FxCore.create.test.ts]: Updated

**Review by tecton — APPROVED** [review]

**Review by yukun-dong — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**